js/projectView.js

```diff
@@ -1,75 +1,74 @@
+(function(module){
 console.log('projectView works');
 var projectView = {};
 
+projectView.handleMainNav = function() {
+  $('nav').on('click', '.tab', function(e) {
+    $('.tab-content').hide();
+    var $selectedTab = $(this).attr('data-content');
+    $('#' + $selectedTab).fadeIn(300);
+  });
+
+  $('nav .tab:nth-of-type(2)').click();
+};
+
+
 projectView.populateFilters = function() {
-  $('article').each(function() {
-    if (!$(this).hasClass('template')) {
-      var val = $(this).find('address a').text();
+  $('div[data-category]').each(function() {
+    if(!$(this).hasClass('template')) {
+      var val = $(this).attr('data-category');
       var optionTag = '<option value="' + val + '">' + val + '</option>';
       $('#rating-filter').append(optionTag);
-
-      val = $(this).attr('data-category');
-      optionTag = '<option value="' + val + '">' + val + '</option>';
       if ($('#category-filter option[value="' + val + '"]').length === 0) {
         $('#category-filter').append(optionTag);
       }
     }
   });
 };
 
-projectView.handleAuthorFilter = function() {
+
+projectView.handleRatingFilter = function() {
   $('#rating-filter').on('change', function() {
     if ($(this).val()) {
-
       $('article').hide();
-      $('article').filter('article[data-developer="' + $(this).val() + '"]').fadeIn(300);
-
+      $('article[data-category="' + $(this).val() + '"]').fadeIn(300);
     } else {
-
       $('article').fadeIn(300);
-      $('.template').hide();
+      $('article.template').hide();
     }
     $('#category-filter').val('');
   });
 };
 
-projectView.handleCategoryFilter = function() {
+
+
+
+
+
+projectView.handleTitleFilter = function() {
   $('#category-filter').on('change', function() {
     if ($(this).val()) {
-      $('article').hide();
-      $('article').filter('article[data-category="' + $(this).val() + '"]').fadeIn(300);
-
+      $('div[data-category]').hide();
+      var $selectedCategory = $('div[data-category=' + $(this).val() + ']');
+      $selectedCategory.show();
     } else {
-      $('article').fadeIn(300);
-      $('.template').hide();
+      $('div[data-category]:not(.template)').show();
     }
-    $('#rating-filter').val('');
   });
 };
 
-projectView.handleMainNav = function() {
-  $('.main-nav').on('click', '.tab', function() {
-    $('.tab-content').hide();
-    $('#' + ($(this).data('content'))).fadeIn(300);
+projectView.initIndexPage = function(){
+  Project.all.forEach(function(instantiatedProject){
+    $('#projects').append(instantiatedProject.toMyPortfolio());
   });
```

Comment:
You're not exposing projectView to the window, so you get an error when you try to pass projectView.initIndexPage as a callback in index.html. Remember, you have to put module.projectView = projectView at the bottom to expose the methods to the window.